### PR TITLE
Show mentees under the mentor profile

### LINKED ISSUE #150: Show mentees under the mentor profile
**Is your feature request related to a problem? Please describe.**
Currently we can't view the mentees under a mentor in the public mentor page

**Describe the solution you'd like**
Show mentees under the mentor profile

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Madhawa97**
## Purpose

The purpose of this PR is to fix #150, #151

## Goals

Show mentees under the mentor profile

## Approach

Show mentees of a mentor as a avatar stack.

## Screenshots

image
image

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by anjula-sack** [comment]
It's not just the profile picture. A mentee card should be shown just like we show for the mentors. @Madhawa97 clicking a mentee card should redirect to a public mentee page

**Comment by Madhawa97** [comment]
@anjula-sack, Also I've added the pending application count as follows,
 
image

File: src/components/ProfilePic/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      alt={alt}
+      className="inline-block rounded-full ring-2 ring-white"
+      style={{ height: size, width: size }}
+      referrerPolicy="no-referrer"
+      onError={() => {
+        setIsError(true);
+      }}
+    />
+  );
+};
```
Comment (anjula-sack) [inline src/components/ProfilePic/index.tsx]:
```suggestion
  
  if(isError || !src) {
   return (
    <div
      style={{ height: size, width: size }}
      className="inline-block bg-gray-200 rounded-full ring-2 ring-white flex items-center justify-center"
    >
      <UserIcon />
    </div>
  ) 
  }

  return (
    <img
      src={src}
      alt={alt}
      className="inline-block rounded-full ring-2 ring-white"
      style={{ height: size, width: size }}
      referrerPolicy="no-referrer"
      onError={() => {
        setIsError(true);
      }}
    />
  );
};
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — APPROVED** [review]
Good job! @Madhawa97

**Comment by Madhawa97** [comment]
Thanks! @anjula-sack

**Pull request merged**